src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -231,7 +231,7 @@ You can define rules to [redirect users to permanently-moved pages](#configured-
 
 You can specify a mapping of permanent redirects in your Astro config with the `redirects` value. For most redirects, this is a mapping of an old route to the new route:
 
-```js title="astro.config.mjs" {5, 7-9}
+```js title="astro.config.mjs" {5}
```

Comment:
```suggestion
```js title="astro.config.mjs" {4-6}
```
I think the original intention was to show also adding the `redirects: {}` config option, since it won't otherwise exist and this is the first time they're seeing this. So, I suggest highlighting all of this!

Reply:
I could see using `ins` for the second instance, comparing `'/new-page'` to the object. In this first case, though, we're not really comparing anything. Thoughts?

Reply:
I see, I was thinking that since this code snippet introduces a new configuration option to the config file. It would be easier to follow the green highlight. Sort of "Add this to the config file to opt-in for redirects"